hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,20 @@
+RELEASE_TYPE: patch
+
+This patch improves support for the SupportsOp protocols from the 
+``typing`` when using on :func:`~hypothesis.strategies.from_type` as outlined in
```

Comment:
:mod:`python:typing`
    :class:`python:typing.SupportsAbs`
    :func:`python:abs`

It's kinda annoying to type, but using cross-reference syntax for everything provides nice hyperlinks for users, and typo-detection for us :smile:

Reply:
oh cool that's great. just didn't know how much i could reference or if it was just internals. will do this.